common/build/eslint-config-fluid/rushstack.js

```diff
@@ -0,0 +1,155 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// This is a workaround for https://github.com/eslint/eslint/issues/3458
+require("@rushstack/eslint-config/patch/modern-module-resolution");
```

Comment:
I was wondering why I didn't see a new dependency for `@rushstack/eslint-config`, but I bet you already have one for "modern-module-resolution"...